Gemfile

```diff
@@ -6,6 +6,8 @@ source 'https://rubygems.org'
 # VMDB specific gems
 #
 
+gem 'ci_reporter_rspec'
```

Comment:
Is there some purpose you had in mind for generating XML output beyond your own personal use? We should discuss that, if so.

If not, yes, this should certainly be removed.